editor.planx.uk/src/theme.ts

```diff
@@ -45,33 +72,11 @@ export const linkStyle = (primaryColor?: string) => ({
   "&:focus-visible": focusStyle,
 });
 
-/**
- * Get Global theme options
- * The global theme is used in editor, and as the base theme in Preview/Unpublished which can be
- * merged with Team specific options
- */
-export const getGlobalThemeOptions = (): ThemeOptions => {
+const getThemeOptions = (primaryColor: string): ThemeOptions => {
   const palette = createPalette({
+    ...DEFAULT_PALETTE,
     primary: {
```

Comment:
This is a shallow merge and unless I'm mistaken it's overwriting `primary.contrastText` into `undefined`.

Reply:
Good catch thank you - you're correct. Fixed with a `deepmerge` 👌

Reply:
Good :+1: